apps/polarion/polarion_set_automated.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+from simple_logger.logger import get_logger
+
+import click
+
+from apps.polarion.polarion_utils import get_polarion_project_id, find_polarion_ids, update_polarion_ids
+
+LOGGER = get_logger(name=__name__)
+
+
+@click.command()
+@click.option(
+    "--config-file-path",
+    help="Provide absolute path to the config file. Any CLI option(s) would override YAML file",
+    type=click.Path(),
+    default=os.path.expanduser("~/.config/python-utility-scripts/config.yaml"),
+)
+@click.option("--project-id", "-p", help="Provide the polarion project id")
+@click.option("--verbosity", default=False, is_flag=True)
+def polarion_approve_automate(config_file_path, project_id, verbosity):
+    if verbosity:
+        LOGGER.setLevel(logging.DEBUG)
```

Comment:
```suggestion
    if not verbose:
        logging.disable(logging.CRITICAL)
```

and then you do not need debug